### DISCUSSION TITLE
Default Thermal properties of air and soot

### DISCUSSION THREAD

**Discussion by NhatNguyenEBC** [firemodels/fds#13727] [Q&A]
Hello everyone,

We are currently comparing results between a Fluent simulation and those of an FDS simulation and have noticed slight discrepancies in the densities/temperatures. In order to improve our comparison, we would like to compare the thermal properties of the fluids (air and soot) before we draw any conclusions. Unfortunately, we were unable to find the reference for the thermal properties of the fluids. We are currently investigating density/specific heat/thermal conductivity and would like to find the underlying equations and values that are implemented for air and soot in FDS. Can someone tell me the source/reference that is used to calculate those properties?

Thank you very much for your support.
Best Regards

**Comment by rmcdermo** [comment]
Have a look at section 3.3.5 Sensible Enthalpy and section 4.2.8 Transport Coefficients for Direct Numerical Simulation (DNS) in the [FDS Tech Guide](https://github.com/firemodels/fds/releases/download/FDS-6.9.1/FDS_Technical_Reference_Guide.pdf) and the references it mentions.  Note that soot is treated as gaseous carbon for the purpose of computing the local fluid density.

**Comment by NhatNguyenEBC** [comment]
Thank you for providing the references. However, based on the given sources for the material properties, I am unable to understand why the measured density of air is not 1.204 $\frac{kg}{m^{3}}$ at t = 0 s. At t = 0 s, my temperature is 20 °C and the ambient pressure is 101325 Pa which should result in a density of 1.204 $\frac{kg}{m^{3}}$ for air. However, the device measures a density of 1.1932  $\frac{kg}{m^{3}}$. Do you have any insight into why there is a small discrepancy between the simulation and the reference data?

Test Case (Cube with devices for density):
[Test.txt](https://github.com/user-attachments/files/17735041/Test.txt)

**Reply by drjfloyd** [reply]
Look at the .out file. It details the composition of AIR and the ambient density.
